web_src/js/features/common-global.js

```diff
@@ -343,6 +343,12 @@ export function initGlobalButtons() {
       window.location.href = $this.attr('data-done-url');
     });
   });
+
+  // get raw text for copy content button
+  const text = Array.from(document.querySelectorAll('.lines-code')).map((el) => el.textContent).join('');
+  for (const copyContentButton of document.querySelectorAll('button.copy-content')) {
+    copyContentButton.setAttribute('data-clipboard-text', text);
+  }
```

Comment:
```suggestion
  const copyContentButtons = document.querySelectorAll('button.copy-content');
  if (copyContentButtons.length == 0) return;
  const text = Array.from(document.querySelectorAll('.lines-code')).map((el) => el.textContent).join('');
  for (const copyContentButton of copyContentButtons) {
    copyContentButton.setAttribute('data-clipboard-text', text);
  }
```
1. I don't think it makes sense to always perform this rather heavy operation, even if the button isn't present
2. This piece of code will fail the second there is a copy content button and more than one displayed file, or even the diff of that file. As long as it is only used on the file view: fine.
Otherwise: not fine.